src/main/java/com/neuvector/Scanner.java

```diff
@@ -63,10 +52,8 @@ public static ScanRepoReportData scanRegistry(Registry registry, NVScanner nvSca
         }else{
             DockerRunCommandBuilder builder = new DockerRunCommandBuilder(nvScanner.getRuntime());
             builder
-                .withUserAndGroup(getDockerUserGroupCmdArg(getScanReportPath(nvScanner.getNvMountPath())))
-                .withName(generateScannerName())
+                .withEntrypoint("/usr/bin/bash")
                 .withVolume(nvScanner.getSocketMapping())
-                .withVolume(appendBindMountSharedSuffixIfRequired(nvScanner.isBindMountShared(), getMountPath(nvScanner), log))
```

Comment:
This is the main change. We do not have the mount path anymore. We do not need it. Same with `appendBindMountSharedSuffixIfRequired`.. This was the `:z` we were adding for some special case some customers were using.